types/@hummer/@hummer-tests.ts

```diff
@@ -0,0 +1,13 @@
+import Hummer from './emulator/Hummer';
```

Comment:
1. filename should be hummer-tests.ts
2. It looks like only ./emulation/Hummer has exports, and the rest of the package is global. But the tests should test as much as possible, so I'd recommend testing the global `Hummer` object with no imports.

Reply:
Hummer is a global API injected by an iOS/Android render library, so there is no such context to test it on Node.js environment. I am wondering how to test those APIs without an environment. Could you help me to figure out the proper way?